src/combat.cpp

```diff
@@ -774,14 +776,14 @@ void Combat::doCombatHealth(Creature* caster, Creature* target, CombatDamage& da
 	}
 
 	if (canCombat) {
+		if (caster && params.distanceEffect != CONST_ANI_NONE) {
+			addDistanceEffect(caster, caster->getPosition(), target->getPosition(), params.distanceEffect);
+		}
+		
```

Comment:
Ahahaha no problem, I was just curious. Kudos to you spreading the version control :D